apps/web/src/services/project/components/ProjectMemberTab.vue

```diff
@@ -74,44 +92,37 @@ const state = reactive({
         },
     ] as MenuItem[])),
     memberInviteFormVisible: false,
-});
-const checkMemberDeleteState = reactive({
-    fields: computed(() => [
-        { name: 'resource_id', label: 'User ID' },
-        { name: 'role_info.name', label: 'Role' },
-    ]),
-    visible: false,
+    memberDeleteModalVisible: false,
 });
 
 /* Api */
-const listMembers = async () => {
+const getProjectUserData = async () => {
     state.loading = true;
     state.selectIndex = [];
     try {
-        const res = await SpaceConnector.client.identity.project.member.list({
+        const params: ProjectGetRequestParams = {
+            workspace_id: '', // TODO: workspace_id
```

Comment:
suggestion: I think it's not needed because workspace_id will be added by SpaceConnector level.